src/main/java/Validate.java

```diff
@@ -0,0 +1,46 @@
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.Set;
+
+public class Validate {
+    public void isValidCarNames(String[] carNames) {
+        checkBlank(carNames);
+        checkLength(carNames);
+        checkDouble(carNames);
+    }
+
+    public void checkDigits(String input) {
+        boolean isMatch = input.matches("\\d+");
+        if (!isMatch) {
+            throw new IllegalArgumentException("[ERROR] : 숫자만 입력이 가능합니다.");
+        }
+    }
+
+    public void checkRange(int tryCount) {
+        if (tryCount < 1) {
+            throw new IllegalArgumentException("[ERROR] : 1이상의 정수만 가능합니다.");
+        }
+    }
+
+    private void checkBlank(String[] carNames) {
+        if (carNames.length == 0) {
+            throw new IllegalArgumentException("[ERROR] : 빈 이름은 들어올 수 없습니다");
+        }
+    }
+
+    private void checkLength(String[] carNames) {
+        long invalidCount = Arrays.stream(carNames)
+                .filter(names -> names.isEmpty() || names.length() > 5)
+                .count();
+        if (invalidCount != 0) {
+            throw new IllegalArgumentException("[ERROR] : 이름 길이는 1~5자 이하여야 합니다");
+        }
+    }
+
+    private void checkDouble(String[] carNames) {
+        Set<String> set = new HashSet<>(Arrays.asList(carNames));
```

Comment:
- 그냥 set이라는 이름보다는 좀 더 명확한 이름이 좋을 것 같아요. carNameSet 이라든지..?
```suggestion
        Set<String> carNameSet = new HashSet<>(Arrays.asList(carNames));
```
- nit : 중복을 제거하는데는 stream의 distinct() 키워드도 있으니 참고해주세요!

Reply:
- 중복 확인하는 임시 변수라 너무 대충 지었던 듯 합니다.. 수정하였습니다
- distinct() 사용법 확인 후 2단계 적용해보겠습니다 새로운 키워드 알려주셔서 감사합니다 !!